adserver/models.py

```diff
@@ -353,6 +354,16 @@ class Advertiser(TimeStampedModel, IndestructibleModel):
     name = models.CharField(_("Name"), max_length=200)
     slug = models.SlugField(_("Advertiser Slug"), max_length=200, unique=True)
 
+    # Publisher specific advertiser account
+    publisher = models.ForeignKey(
```

Comment:
Perhaps a `OneToOneField` is better.

Reply:
Seems likely. Though there might be some weird edge cases where multiple publishers want to run the same house ads? 🤷